xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/resources/document-formats.js

```diff
@@ -19,196 +19,666 @@
-    "extensions": ["pptx"],
+    "name": "Microsoft PowerPoint 2007-2013 XML",
+    "extensions": [
+      "pptx"
+    ],
     "mediaType": "application/vnd.openxmlformats-officedocument.presentationml.presentation",
-    "inputFamily": "PRESENTATION"
+    "inputFamily": "PRESENTATION",
+    // Filter previously missing: keep it?
+    "storeProperties": {
+      "PRESENTATION": {
+        "FilterName": "Impress MS PowerPoint 2007 XML"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Microsoft PowerPoint 2007-2013 XML Template",
+    "extensions": [
+      "potx"
+    ],
+    "mediaType": "application/vnd.openxmlformats-officedocument.presentationml.template",
+    "inputFamily": "PRESENTATION",
+    "storeProperties": {
+      "PRESENTATION": {
+        "FilterName": "Impress MS PowerPoint 2007 XML"
+      }
+    }
+  },
+  {
+    "name": "Microsoft PowerPoint 97-2003",
+    "extensions": [
+      "ppt"
+    ],
+    "mediaType": "application/vnd.ms-powerpoint",
+    "inputFamily": "PRESENTATION",
+    "storeProperties": {
+      "PRESENTATION": {
+        "FilterName": "MS PowerPoint 97"
+      }
+    }
+  },
+  {
+    "name": "OpenOffice.org 1.0 Presentation",
+    "extensions": [
+      "sxi"
+    ],
+    "mediaType": "application/vnd.sun.xml.impress",
+    "inputFamily": "PRESENTATION",
+    "storeProperties": {
+      "PRESENTATION": {
+        "FilterName": "StarOffice XML (Impress)"
+      }
+    }
   },
   {
     "name": "OpenDocument Drawing",
-    "extensions": ["odg"],
+    "extensions": [
+      "odg"
+    ],
     "mediaType": "application/vnd.oasis.opendocument.graphics",
     "inputFamily": "DRAWING",
-    "storeProperties": {"DRAWING": {"FilterName": "draw8"}}
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw8"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "OpenDocument Drawing Template",
+    "extensions": [
+      "otg"
+    ],
+    "mediaType": "application/vnd.oasis.opendocument.graphics-template",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw8_template"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "OpenDocument Drawing Flat XML",
+    "extensions": [
+      "fodg"
+    ],
+    "mediaType": "application/vnd.oasis.opendocument.graphics-flat-xml",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "OpenDocument Drawing Flat XML"
+      }
+    }
+  },
+  {
+    "name": "Portable Document Format",
+    "extensions": [
+      "pdf"
+    ],
+    "mediaType": "application/pdf",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_pdf_Export"
+      },
+      "SPREADSHEET": {
+        "FilterName": "calc_pdf_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_pdf_Export"
+      },
+      "TEXT": {
+        "FilterName": "writer_pdf_Export"
+      },
+      // New property
+      "WEB": {
```

Comment:
So this wasn't there in our previous version of the file, I don't think it's harmful to have it.